tests/compiler/test_null.py

```diff
@@ -41,39 +41,6 @@ def test_null(snippetcompiler):
     assert isinstance(a, NoneValue)
 
 
-def test_null_unset(snippetcompiler):
-    snippetcompiler.setup_for_snippet(
-        """
-        entity A:
-            string? a
-        end
-        implement A using std::none
-        a = A()
-
-    """
-    )
-
-    (_, scopes) = compiler.do_compile()
-    root = scopes.get_child("__config__")
-    with pytest.raises(OptionalValueException):
-        root.lookup("a").get_value().get_attribute("a").get_value()
-
-
-def test_null_unset_hang(snippetcompiler):
-    snippetcompiler.setup_for_snippet(
-        """
-            entity A:
-                string? a
-            end
-            implement A using std::none
-            a = A()
-            b = a.a
-        """
-    )
-    with pytest.raises(OptionalValueException):
-        (_, scopes) = compiler.do_compile()
-
-
```

Comment:
I think those 2 tests can just be removed?

Reply:
I would keep the first and update the check to assert that the value is `null` rather than expect the exception.

Reply:
you mean keep `test_null_unset`
and changing `string? a` to `string? a = null` and then verifying it is null? 
cause the test just above(`test_null`) test exactly that.

Reply:
Perfect, in that case I withdraw my comment.